### PR TITLE
removes adobe tag manager and pepperjam.com

### PR DESCRIPTION
**Pull request by patjack**
Adds the following commits only:

https://github.com/disconnectme/disconnect-tracking-protection/commit/f39cf6021449f3ce1940301aa48ee3c37514de14

https://github.com/disconnectme/disconnect-tracking-protection/commit/c03538dcc03b909c78be1ed660ea938bc6d74c66

### CODE REVIEW

**Comment by englehardt** [comment]
@patjack would you mind to move https://github.com/disconnectme/disconnect-tracking-protection/commit/ea5b9beb3d671dff4e09c4283f1717e7f5d1de9b from #76 to this PR? That change will fix some Firefox breakage, and we'd like to merge it before we merge #76.

**Comment by patjack** [comment]
@englehardt no problem. I just pushed the change to this PR.

**Comment by Brandr0id** [comment]
@englehardt thoughts on pulling in [disconnectme/disconnect-tracking-protection@ 014f137](https://github.com/disconnectme/disconnect-tracking-protection/commit/014f1374e47cb52e23419d0a7d56d820b4b60650) as well since it addresses some breakage w/Mercadolibre?

**Comment by stpeter** [comment]
@Brandr0id AIUI we're trying to avoid additions right now because of the versioning work, but deletions are safe.

**Comment by stpeter** [comment]
@Brandr0id Sorry, I mis-spoke, this should be fine. I'll leave such things to @englehardt in the future. :-)

**Comment by groovecoder** [comment]
I went ahead and merged this as-is, for the original 2 requests. I'm watching the list creation job, which should run in about 4m.

File: disconnect-blacklist.json [hunk 1]
```diff
@@ -10967,6 +10959,7 @@
       {
         "Facebook": {
           "http://www.facebook.com/": [
+            "apps.fbsbx.com",
```
Comment (groovecoder) [inline disconnect-blacklist.json]: `ValueError: Blocklist contains block rule apps.fbsbx.com under the Disconnect category, but the rule is not found in the given Disconnect mapping file.`

Updating ...

Comment (groovecoder) [reply disconnect-blacklist.json]: @englehardt https://github.com/mozilla-services/shavar-list-creation/pull/101

**Pull request merged**